pkg/controller/logstorage/logstorage_controller_test.go

```diff
@@ -535,3 +540,15 @@ func setUpLogStorageComponents(cli client.Client, ctx context.Context, storageCl
 		}),
 	).ShouldNot(HaveOccurred())
 }
+
+func (*mockESClient) NewElasticsearchClient(client.Client, context.Context) error {
+	return nil
+}
+
+func (*mockESClient) SetElasticsearchIlmPolicies(context.Context, *operatorv1.LogStorage, int64) error {
```

Comment:
I think ILM should be three capital letters. In golang abbreviations are all uppercase when the three letters stand for three separate words. (Like URL for example)